Gemfile

```diff
@@ -55,13 +55,12 @@ group :system_tests do
   gem "puppet-module-posix-system-r#{minor_version}",                            :require => false, :platforms => "ruby"
   gem "puppet-module-win-system-r#{minor_version}",                              :require => false, :platforms => ["mswin", "mingw", "x64_mingw"]
   gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '>= 3')
```

Comment:
Could also do this by changing `>= 3` to `~> 3.37`.

Reply:
My original thought process was that explicitly pinning back the dependency that blocked beaker 4 (beaker-rspec) would reflect the incremental progress to beaker 4 compatibility. For example specifically requiring the beaker-[hypervisor] libraries. Pinning beaker seems to be more straight forward though. Updated.